Neos.ContentRepository.Core/Classes/CommandHandler/CommandBus.php

```diff
@@ -35,9 +36,13 @@ public function handle(CommandInterface $command): EventsToPublish|\Generator
     {
         // multiple handlers must not handle the same command
         foreach ($this->handlers as $handler) {
-            if ($handler->canHandle($command)) {
-                return $handler->handle($command, $this->commandHandlingDependencies);
+            if (!$handler->canHandle($command)) {
+                continue;
             }
+            foreach ($this->commandHooks as $commandHook) {
+                $command = $commandHook->onBeforeHandle($command);
+            }
+            return $handler->handle($command, $this->commandHandlingDependencies);
         }
```

Comment:
regarding _which_ commands are passed to `onBeforeHandle`:

Right now `SetProperties` would be passed but NOT `SetSerializedProperties`.
I have a change which would change that and would like to discuss that either way:)

Reply:
Why is SetSerializedNodeProperties not handled?

Reply:
> Yes, let's discuss

We _did_ and the outcome is: [SetSerializedNodeProperties](https://github.com/neos/neos-development-collection/blob/b486a64112c311916344f79fbc344ae8849e092e/Neos.ContentRepository.Core/Classes/Feature/NodeModification/Command/SetSerializedNodeProperties.php#L33) should not only be _marked_ internal but it should not even implement the `CommandInterface`.
Instead we want to add some low-level concept for those "serializable commands"..

For this PR it means: we don't care about those :)

Reply:
FYI: will be addressed with #5348